.circleci/config.yml

```diff
@@ -598,6 +598,8 @@ jobs:
           name: Save Vault secrets into temp file to persist across jobs
           command: |
             set -u
+            echo "export ENV=<< parameters.env >>" >> $BASH_ENV
+            source $BASH_ENV
```

Comment:
Create a new env variable `ENV` equals to `<< parameters.env >>` so `export-playwright-env.sh` can use the env variable.